app/components/shared/work_history_item_component.rb

```diff
@@ -21,8 +21,11 @@ def title
   end
 
   def details
-    return item.details if item.respond_to?(:details)
     return item.reason if item.respond_to?(:reason)
```

Comment:
I don't think the order of these mattered, since it's either a gap in history with a `reason`, or a work/volunteering experience with a `details`.